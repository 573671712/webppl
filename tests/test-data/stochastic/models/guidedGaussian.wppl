var model = function() {
  var a = sample(Gaussian({mu: -2, sigma: 1.5}), {
    guide: Gaussian({
      mu: param(0),
      sigma: Math.exp(param(0))
    })
  });
  return a;
};
