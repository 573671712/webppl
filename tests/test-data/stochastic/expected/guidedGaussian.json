{
  "mean": -2,
  "std": 1.5
}
